solution/quiz 4

```diff
@@ -0,0 +1,27 @@
+import csv
+f=open('Salary.csv','w')
```

Comment:
i can not under staned this line??

Reply:
this line is used to open your csv file for write.